x-pack/legacy/plugins/ml/public/data_frame/pages/job_management/page.tsx

```diff
@@ -21,45 +23,58 @@ import {
   EuiTitle,
 } from '@elastic/eui';
 
+import { useRefreshTransFormList } from '../../common';
 import { CreateJobButton } from './components/create_job_button';
 import { DataFrameJobList } from './components/job_list';
+import { RefreshTransformListButton } from './components/refresh_transform_list_button';
 
-export const Page: SFC = () => (
-  <EuiPage data-test-subj="mlPageDataFrame">
-    <EuiPageBody>
-      <EuiPageContentHeader>
-        <EuiPageContentHeaderSection>
-          <EuiTitle>
-            <h1>
-              <FormattedMessage
-                id="xpack.ml.dataframe.jobsList.dataFrameTitle"
-                defaultMessage="Data frame jobs"
-              />
-              <span>&nbsp;</span>
-              <EuiBetaBadge
-                label={i18n.translate('xpack.ml.dataframe.jobsList.betaBadgeLabel', {
-                  defaultMessage: `Beta`,
-                })}
-                tooltipContent={i18n.translate(
-                  'xpack.ml.dataframe.jobsList.betaBadgeTooltipContent',
-                  {
-                    defaultMessage: `Data frames are a beta feature. We'd love to hear your feedback.`,
-                  }
-                )}
-              />
-            </h1>
-          </EuiTitle>
-        </EuiPageContentHeaderSection>
-        <EuiPageContentHeaderSection>
-          <CreateJobButton />
-        </EuiPageContentHeaderSection>
-      </EuiPageContentHeader>
-      <EuiPageContentBody>
-        <EuiSpacer size="l" />
-        <EuiPanel>
-          <DataFrameJobList />
-        </EuiPanel>
-      </EuiPageContentBody>
-    </EuiPageBody>
-  </EuiPage>
-);
+export const Page: SFC = () => {
+  const { isLoading, refresh } = useRefreshTransFormList();
+
+  return (
+    <EuiPage data-test-subj="mlPageDataFrame">
+      <EuiPageBody>
+        <EuiPageContentHeader>
+          <EuiPageContentHeaderSection>
+            <EuiTitle>
+              <h1>
+                <FormattedMessage
+                  id="xpack.ml.dataframe.jobsList.dataFrameTitle"
+                  defaultMessage="Data frame jobs"
+                />
+                <span>&nbsp;</span>
+                <EuiBetaBadge
+                  label={i18n.translate('xpack.ml.dataframe.jobsList.betaBadgeLabel', {
+                    defaultMessage: `Beta`,
+                  })}
+                  tooltipContent={i18n.translate(
+                    'xpack.ml.dataframe.jobsList.betaBadgeTooltipContent',
+                    {
+                      defaultMessage: `Data frames are a beta feature. We'd love to hear your feedback.`,
+                    }
+                  )}
+                />
+              </h1>
+            </EuiTitle>
+          </EuiPageContentHeaderSection>
+          <EuiPageContentHeaderSection>
+            <EuiFlexGroup alignItems="center">
+              <EuiFlexItem>
+                <RefreshTransformListButton onClick={() => refresh()} isLoading={isLoading} />
```

Comment:
I think for the `onClick` here you should just be able to pass `refresh` without wrapping it in another function. `onClick={refresh}`

Reply:
Fixed in 0ee6553.